src/vm/jitinterface.cpp

```diff
@@ -3187,72 +3162,199 @@ void CEEInfo::ComputeRuntimeLookupForSharedGenericToken(DictionaryEntryKind entr
             pResult->helper = fInstrument ? CORINFO_HELP_RUNTIMEHANDLE_CLASS_LOG : CORINFO_HELP_RUNTIMEHANDLE_CLASS;
         }
         // If we've got an object, go through its vtable
-        else 
+        else
         {
             _ASSERTE(pContextMD->AcquiresInstMethodTableFromThis());
             pResultLookup->lookupKind.runtimeLookupKind = CORINFO_LOOKUP_THISOBJ;
             pResult->helper = fInstrument ? CORINFO_HELP_RUNTIMEHANDLE_CLASS_LOG : CORINFO_HELP_RUNTIMEHANDLE_CLASS;
         }
+    }
+
```

Comment:
I'll revisit this in the near future when re-enabling the dictionary lookup optimization for generic arg type handle lookups

---
In reply to: [63013661](https://github.com/dotnet/coreclr/pull/4519#discussion_r63013661) [](ancestors = 63013661,63012927)